source/androidx.leanback/leanback-preference/transforms/Metadata.xml

```diff
@@ -1,6 +1,24 @@
 ﻿<metadata>
+    <!--
     <attr path="/api/package[@name='androidx.leanback.preference']/class[@name='LeanbackListPreferenceDialogFragment.AdapterMulti']/method[@name='onCreateViewHolder']" name="return">AndroidX.RecyclerView.Widget.RecyclerView.ViewHolder</attr>
+    -->
     <attr path="/api/package[@name='androidx.leanback.preference']/class[@name='LeanbackListPreferenceDialogFragment.AdapterMulti']/method[@name='onBindViewHolder']/parameter[1]" name="type">AndroidX.RecyclerView.Widget.RecyclerView.ViewHolder</attr>
+    <!--
     <attr path="/api/package[@name='androidx.leanback.preference']/class[@name='LeanbackListPreferenceDialogFragment.AdapterSingle']/method[@name='onCreateViewHolder']" name="return">AndroidX.RecyclerView.Widget.RecyclerView.ViewHolder</attr>
+    -->
```

Comment:
I would suggest just removing these lines instead of commenting them out. Source control has our backs! I hope :)

Reply:
Removed. Blown to pieces